source/testing/testing-components.md

```diff
@@ -1,16 +1,15 @@
-_Unit testing methods and computed properties follows previous patterns shown
-in [Unit Testing Basics] because Ember.Component extends Ember.Object._
 
-Components can be tested using the `moduleForComponent` helper.
+
+Components can be tested with integration tests using the `moduleForComponent` helper.
 
 Let's assume we have a component with a `style` property that is updated
 whenever the value for its `name` property changes. The `style` attribute of the
 component is bound to its `style` property.
 
 > You can follow along by generating your own component with `ember generate
-> component pretty-color`.
+> component pretty-color --pod`.
```

Comment:
We're not going to encourage people to use the pod directory structure until it's format has settled down. Can you re-do the examples with the "regular" structure for now?